server/src/workers/api.ts

```diff
@@ -42,23 +41,18 @@ async function bootstrap() {
   useSwagger(app, { write: isDev });
 
   app.setGlobalPrefix('api', { exclude: excludePaths });
-  if (existsSync(resourcePaths.web.root)) {
-    // copied from https://github.com/sveltejs/kit/blob/679b5989fe62e3964b9a73b712d7b41831aa1f07/packages/adapter-node/src/handler.js#L46
-    // provides serving of precompressed assets and caching of immutable assets
-    app.use(
-      sirv(resourcePaths.web.root, {
-        etag: true,
-        gzip: true,
-        brotli: true,
-        extensions: [],
-        setHeaders: (res, pathname) => {
-          if (pathname.startsWith(`/_app/immutable`) && res.statusCode === 200) {
-            res.setHeader('cache-control', 'public,max-age=31536000,immutable');
-          }
-        },
-      }),
-    );
+
+  if (!isDev) {
```

Comment:
I think this should not be tied to ENV, but instead be tied to the file existing or not. Can we just wrap this import with a try/catch. If the file exists we will use it, otherwise we just log a warning and move on.